### PR TITLE
initialized repo with eslint

### PR DESCRIPTION
**Pull request by tafodinho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "eslint": "^6.3.0",
```
Comment (rvvergara) [inline package.json]: `Eslint` should be in `devDependencies` and not in `dependencies`. And the way we should have installed this is by running in the terminal the ff command (and selecting the appropriate options):
```bash
npx eslint --init
```

Comment (tafodinho) [reply package.json]: I did exactly as you have described above i don't know why i have those dependency issues am currently looking into it

Comment (tafodinho) [reply package.json]: in the place of npx create-react-app react-redux-bookstore i used instead create-react-app react-redux-bookstore without the npx as i followed on the tutorial could that have been the problem ??

Comment (rvvergara) [reply package.json]: That's why I specifically said you should use `npx create-react-app` :)

**Pull request merged**